src/Display.js

```diff
@@ -5,12 +5,12 @@ import ListOfWinners from "./components/ListOfWinners";
 import {QuizContainer} from "./QuizContainer";
 
 function Display() {
-    const {quizStatus, registeredUser} = useContext(QuizContainer.QuizContext);
+    const {quizStatus, winnersList} = useContext(QuizContainer.QuizContext);
     const quizScheduled = quizStatus === "scheduled";
     const gameInProgress = quizStatus === "started";
-    const showListOfWinners = quizStatus === "finished";
+    const showListOfWinners = winnersList.length > 0;
 
-    if(gameInProgress) {
+    if(gameInProgress && !showListOfWinners) {
```

Comment:
w sumie zamiast tego `&& !showListOfWinners` można po prostu przenieść
```
if(showListOfWinners) {
        return <ListOfWinners/>
    }
```
powyżej tego ifa z gameInProgress to wyjdzie na to samo (ale będzie ładniej i prościej)